src/main/java/com/entropyteam/entropay/employees/repositories/HolidayRepository.java

```diff
@@ -2,8 +2,13 @@
 
 import com.entropyteam.entropay.common.BaseRepository;
 import com.entropyteam.entropay.employees.models.Holiday;
+import org.springframework.data.jpa.repository.Query;
 
+import java.util.List;
 import java.util.UUID;
 
 public interface HolidayRepository extends BaseRepository<Holiday, UUID> {
+
+    @Query(value = "SELECT DISTINCT extract('Year' FROM date) AS year FROM holiday_calendar ORDER BY year ASC", nativeQuery = true)
+    List<Integer> getHolidaysYears();
```

Comment:
does this method include deleted holidays? we shouldn't consider the years of deleted holidays. The new query will be:

@Query(value = "SELECT DISTINCT extract('Year' FROM date) AS year FROM holiday_calendar WHERE deleted=false ORDER BY year ASC", nativeQuery = true)
List<Integer> getHolidaysYears();

Reply:
good idea, I didn't think of it, add it, thanks